README.md

```diff
@@ -182,6 +182,56 @@ isValid(fragments); // display false because ISSUER_IDENTITY is INVALID
 isValid(fragments, ["DOCUMENT_INTEGRITY", "DOCUMENT_STATUS"]); // display true because those types are VALID
 ```
 
+## Utils and types
```

Comment:
might want to update readme on Usage section where it's showing the old fragment:

```
  {
    "name": "OpenAttestationDnsTxt",
    "type": "ISSUER_IDENTITY",
    "data": [
      {
        "status": "VALID",
        "location": "example.tradetrust.io",
        "value": "0xe59877ac86c0310e9ddaeb627f42fdee5f793fbe"
      }
    ],
    "status": "VALID"
  }
```

Reply:
change what ?

Reply:
done